README.md

```diff
@@ -0,0 +1,137 @@
+
+# o-fonts [![Build Status](https://travis-ci.org/Financial-Times/o-fonts.svg)](https://travis-ci.org/Financial-Times/o-fonts)
+
+Easily include FT web fonts in products.
+
+## Quick start
+
+```html
+<!-- Load web fonts with @font-face declarations  -->
+<link rel="stylesheet" href="//build.origami.ft.com/bundles/css?modules=o-fonts@^1" />
+
+<!-- Set font families -->
+<style>
+    html {
+        font-family: BentonSans, sans-serif;
+    }
+    h1 {
+        font-family: MillerDisplay, serif;
+    }
+</style>
+```
+
+[Looking for more advanced usage options (Sass…)?](#advanced)
+
+----
+
+## Browser support
+
+`o-fonts` loads web fonts in the [WOFF format](http://en.wikipedia.org/wiki/Web_Open_Font_Format).
+
+WOFF is supported in IE 9+, Chrome, Firefox, iOS 5+, Android 4.4+. [Full support table on caniuse.com](http://caniuse.com/#feat=woff).
+
+## Font families and variants
+
+| BentonSans | MillerDisplay | MetricWeb | FinancierDisplayWeb | FinancierTextWeb |
+|:-:|:-:|:-:|:-:|:-:|
+| | | thin |
+| light | | light |
+| regular | regular | regular | regular | regular |
+| | | regular (italic) | regular (italic) | regular (italic) |
+| | | medium |
+| | | semibold |
+| bold | bold | bold |
+| | | bold (italic) |
+| | black |
```

Comment:
Might be easier to maintain this if you have a column on the left for style and then use unicode ticks and whitespace to fill in the cols:

```
| Style | BentonSans | MillerDisplay | MetricWeb | FinancierDisplayWeb | FinancierTextWeb |
|:-:|:-:|:-:|:-:|:-:|:-:|
| thin  |   |   | ✓ |   |   |   |
| light | ✓ |   | ✓ |   |   |   |
...
```